src/backend/access/external/pxfheaders.c

```diff
@@ -165,6 +167,18 @@ static void add_tuple_desc_httpheader(CHURL_HEADERS headers, Relation rel)
         /* Add a key/value pair for attribute type name */
         resetStringInfo(&formatter);
         appendStringInfo(&formatter, "X-GP-ATTR-TYPENAME%u", i);
+
```

Comment:
This might be not needed anymore since we merged HAWQ-997 and we are sending two new headers X-GP-ATTR-TYPEMODX-COUNT, X-GP-ATTR-TYPEMODX-Y along with all columns which might have modifiers(numeric, varchar, char).